hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/management/targettable/TargetTableHeader.java

```diff
@@ -378,16 +378,10 @@ private Boolean doValidations(final DragAndDropEvent dragEvent) {
         return isValid;
     }
 
+    @SuppressWarnings("unchecked")
```

Comment:
This method is small but I would still move the `@SuppressWarnings("unchecked")` to line `383` where the actual unchecked operation occurs